### DISCUSSION TITLE
[Question] Multiple Backup Actions on a single Volume - 7 daily, 1 weekly, 1 monthly, 1 yearly ...

### DISCUSSION THREAD

**Discussion by thiscantbeserious** [longhorn/longhorn#4540] [Q&A]
I'm wondering - does anyone know if ...

... the indivual backup-actions follow the replaction rules used or are they overriding each other by any chance?

Aas in are backups replication action set per volume or per volume-action? As in does it seperates the individual backups properly?

I want to have 7 daily backups, 1 weekly, 1 monthly and 1 yearly for example

 - so I'd logically setup 4 different backup actions from how I understand it ... correct?

I know logically speaking that since there's the possiblity to add multiple backup/snapshot options which can have different cron-entries it would make sense that its properly seperated but I couldn't find anything related to it.

So I figured I better ask before being faced with just 1 backup while I wanted to have 7/1/1/1 after a year for that specific volume

(specifically since I'm backing it up to a local NFS share, which I plan to duplicate to GDrive with rclone/restic anyway.

**Comment by mantissahz** [comment]
Hi @thiscantbeserious,
Yes, you need to setup 4 `recurring jobs` with type `backup` for an individual volume and you can setup the recurring job by the UI (volume detail page) or a manifest file.
Here is the document: [set-up-recurring-jobs-using-the-longhorn](https://longhorn.io/docs/1.3.1/snapshots-and-backups/scheduling-backups-and-snapshots/#set-up-recurring-jobs-using-the-longhorn-ui)
